src/services/objections.service.ts

```diff
@@ -48,3 +48,13 @@ export const submitObjection = (objectionId: string, token: string) => {
   // TODO Call the Objections SDK. Covered by JIRA
   //      sub-task BI-4143
 };
+
+export const addAttachment = (companyNumber: string,
+                              token: string,
+                              objectionId: string,
+                              attachment: Buffer,
+                              fileName: string) => {
+
+  logger.info(`Adding attachment to objection`);
```

Comment:
would you be able to add more info into the message?  Like:
Adding attachment ${fileName} to objection ${objectionId}

Reply:
I can do.